src/scripts/assets/covid_lab_results_standard.json

```diff
@@ -0,0 +1,18 @@
+{
+    "POSITIVE": [
+        "POSITIVE",
+        "DETECTED",
+        "REACTIVE"
```

Comment:
Dan, just so I understand, these are the state derived from the loinc codes that _definitively_ capture a patient's infection status? Do we know if this list is exhaustive and comprehensive, or are there other resources we need to check to add potential statuses?

Reply:
There is no exhaustive list because it is whatever the labs/hospitals happen to report which is sometimes subject to equipment changes and human error.

Reply:
Mmmm got it. I would love to see us use an ML identification tool to pull things like that out, but that's a ways down the road so I'm perfectly happy with this for the time being.